packages/client/src/views/GrantDetails.vue

```diff
@@ -1,5 +1,13 @@
 <template>
   <section>
+    <div class="container-fluid">
```

Comment:
Sorry, I don't know if I was clear enough in my last comment. I think the HTML and CSS can be much cleaner if we utilize the existing CSS grid setup on the [`<div class="grant-details-container">` div](https://github.com/usdigitalresponse/usdr-gost/pull/2710/files#diff-456776fc1d1394d51d55789b5ae9f202b288850b0586d36f58f3c4330bd44a9eR18). 

Specifically: 

1. We should no longer need the `<div class="container-fluid">` div on this line. 
2. The two `<div class="grant-details-back-link">` elements would move inside the `<div class="grant-details-container">`. 
3. The CSS edits you have right now (e.g., messing with the padding) can be removed. 
4. Instead, you'd just need to update the grid with something like this: 

```diff
diff --git a/packages/client/src/views/GrantDetails.vue b/packages/client/src/views/GrantDetails.vue
index 5b4f3100..71009e9c 100644
--- a/packages/client/src/views/GrantDetails.vue
+++ b/packages/client/src/views/GrantDetails.vue
@@ -445,11 +445,15 @@ export default {
   grid-template-columns: 1fr 437px;
   grid-template-rows: auto;
   grid-template-areas:
+    "back-link back-link"
     "headline  main-actions"
     "content   secondary-actions";
   column-gap: 90px;
   row-gap: 48px;
 }
+.grant-details-back-link {
+  grid-area: back-link;
+}
 .grant-details-headline {
   grid-area: headline;
   align-self: end;
```

Hopefully that makes it clearer what I had in mind, but let me know if it'd be helpful to discuss live!

Reply:
Include the back inside the container and the grid. I was thinking is there any situation where the details content doesn't render we won't have a back button. What you think?